src/Features/LanguageServer/Protocol/Handler/SourceGenerators/SourceGeneratedFileGetTextHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Roslyn.Utilities;
+using LSP = Roslyn.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+[ExportCSharpVisualBasicStatelessLspService(typeof(SourceGeneratedFileGetTextHandler)), Shared]
+[Method("sourceGeneratedFile/_roslyn_getText")]
```

Comment:
nit, my preference is `_roslyn_` as the prefix of the entire method.  but htis is fine as well.

Reply:
We do this pattern everywhere else. I don't know why (it confuses me too) -- @dibarbet do you know the backstory?